core/src/main/scala/com/evolutiongaming/kafka/flow/PartitionFlow.scala

```diff
@@ -189,8 +192,10 @@ object PartitionFlow {
 
       clock <- Clock[F].instant
       timestamp <- timestamp updateAndGet (_.copy(clock = clock))
+
       states <- cache.values
-      _ <- states.values.toList.parTraverse_ { state =>
+      cores <- Runtime[F].availableCores
```

Comment:
No reason to do this on every batch and timer triggering. Let's make it a part of creating the instance (`init` used in `val acquire`). We could multiply it once and pass `parallelism: Int` to `def processRecords` and `def triggerTimers`